config/packages/framework.yaml

```diff
@@ -9,6 +9,7 @@ framework:
         handler_id: ~
         cookie_secure: auto
         cookie_samesite: lax
+        name: SULUSESSID # This avoid conflicts to other application feel free to remove this to set it to default
```

Comment:
`This avoids conflicts with other applications`, and I don't know what exactly you want to say with the `feel free` part, I think I would just remove that last part...

Reply:
And isn't that something we should do in sulu/sulu as well? If somebody is running the application directly there he might run into the same problems.

Reply:
sulu/sulu pull request done